robot_smach_states/src/robot_smach_states/navigation/control_to_pose.py

```diff
@@ -0,0 +1,136 @@
+#
+# Copyright (c) 2019, TU/e Robotics, Netherlands
+# All rights reserved.
+#
+# \author Rein Appeldoorn
+
+import math
+from collections import namedtuple
+
+# ROS
+from geometry_msgs.msg import PoseStamped, Twist, Vector3
+import rospy
+import smach
+import tf2_geometry_msgs
+import tf2_ros
+from tf.transformations import euler_from_quaternion
+
+# TU/e Robotics
+from robot_smach_states.util.geometry_helpers import wrap_angle_pi
+from robot_smach_states.util.designators.checks import check_type
+
+_ = tf2_geometry_msgs  # tf2_geometry_msgs must be declared here for it to be imported
+
+
+def _clamp(abs_value, value):
+    return max(-abs_value, min(abs_value, value))
+
+
+def _get_yaw_from_quaternion_msg(msg):
+    """
+    Returns the yaw angle from a rotation in quaternion representation (msg)
+    :param msg: The quaternion msg
+    :return: Yaw angle
+    """
+    orientation_list = [msg.x, msg.y, msg.z, msg.w]
+    _, _, yaw = euler_from_quaternion(orientation_list)
+    return yaw
+
+
+class ControlParameters(namedtuple('ControlParameters', [
+    'position_gain',
+    'rotation_gain',
+    'abs_vx',
+    'abs_vy',
+    'abs_vyaw',
+    'goal_position_tolerance',
+    'goal_rotation_tolerance'
+])):
+    """
+    position_gain: (float) Tunable parameter to increase or decrease the change in position per time
+    rotation_gain: (float) Tunable parameter to increase or decrease the change in yaw per time
+    abs_vx: (float) Absolute velocity in x
+    abs_vy: (float) Absolute velocity in y
+    abs_vyaw: (float) Absolute velocity in yaw
+    goal_position_tolerance: (float) Tolerance in position
+    goal_rotation_tolerance: (float) Tolerance in yaw
+    """
```

Comment:
Indent these lines by one more tab?

Reply:
Also, units are missing

Reply:
no more indentation according to pycharm pep8 checking, units added